org.civicrm.bootstrapcivicrm/bootstrapcivicrm.php

```diff
@@ -130,6 +130,7 @@ function bootstrapcivicrm_civicrm_pageRun($page) {
     CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.bootstrapcivicrm', 'css/bootstrap.css');
     CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrapcivicrm', 'js/radio-checkbox.js');
     CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrapcivicrm', 'js/add-missing-date-addons.js');
+    CRM_Core_Resources::singleton()->addScriptFile('org.civicrm.bootstrapcivicrm', 'js/civihr-popup.js');
```

Comment:
I'm not sure this scripts belong to the bootstrap extension, i think it makes more sense to have it in `hrui`